pkgs/development/python-modules/pygeos/default.nix

```diff
@@ -17,6 +17,12 @@ buildPythonPackage rec {
     sha256 = "sha256-PEFULvZ8ZgFfRDrj5uaDUDqKIh+cJPsjgPauQq7RYAo=";
   };
 
+  patches = [
+    # Adapt https://github.com/shapely/shapely/commit/4889bd2d72ff500e51ba70d5b954241878349562,
+    # backporting to pygeos
+    ./fix-for-geos-3-11.patch
```

Comment:
If it's just a matter of extracting a single file's diff from the commit, you can just do

```
(fetchpatch {
  name = "fix-tests-geos-3.11.patch";
  url = "https://github.com/shapely/shapely/commit/4889bd2d72ff500e51ba70d5b954241878349562.patch";
  includes = [ "shapely/tests/test_constructive.py" ];
  sha256 = "...";
})
```